newIDE/app/src/Profile/Subscription/SubscriptionDetails.js

```diff
@@ -123,7 +124,8 @@ const SubscriptionDetails = ({
               }
               actions={[
                 !redemptionCodeExpirationDate &&
-                !hasMobileAppStoreSubscriptionPlan(subscription) ? (
+                !hasMobileAppStoreSubscriptionPlan(subscription) &&
+                !hasSubscriptionBeenManuallyAdded(subscription) ? (
```

Comment:
I wonder though if the backend could not handle this case?

Reply:
Probably, and yet this feature doesn't make sense for them, so rather hide it.
What should it do if the BE handles it? 🤔